closed/OpenJ9.gmk

```diff
@@ -419,7 +419,9 @@ endif
 	@$(CP) -p $(OUTPUT_ROOT)/vm/redirector/$(LIBRARY_PREFIX)jvm_b156$(SHARED_LIBRARY_SUFFIX) $(MODULES_LIBS_DIR)/java.base/server/$(LIBRARY_PREFIX)jvm$(SHARED_LIBRARY_SUFFIX)
 
 ifeq (true,$(OPENJ9_ENABLE_DDR))
-	@$(ECHO) Copying j9ddr.dat
+	@$(ECHO) Running ddrgen to create j9ddr.dat, then copying it to lib directory
+	$(MAKE) -C $(OUTPUT_ROOT)/vm/ddr -f run_omrddrgen.mk VERSION_MAJOR=9 OPENJDK_VERSION_NUMBER_FOUR_POSITIONS=$(VERSION_NUMBER_FOUR_POSITIONS) \
+		CC="$(CC)" CXX="$(CXX)" $(EXPORT_MSVS_ENV_VARS)
```

Comment:
This work should be captured in a rule (as in https://github.com/ibmruntimes/openj9-openjdk-jdk8/pull/110) so this work only happens as necessary.